### PR TITLE
[Feature] Set OFPT_FLOW_MOD and OFPT_BARRIER_REQUEST message priority

### PR DESCRIPTION
**Pull request by viniarck**
### Description of the change

- Set KytosEvent priority for OFPT_FLOW_MOD and OFPT_BARRIER_REQUEST

This PR makes use of the `msg_out` queue priority infrastructure that's being delivered [on this kytos core PR](https://github.com/kytos-ng/kytos/pull/226), and it also depends on [this of_core PR](https://github.com/kytos-ng/of_core/pull/66)

### CODE REVIEW

File: requirements/dev.txt [hunk 1]
```diff
@@ -4,10 +4,10 @@
 #
 #    pip-compile --output-file=requirements/dev.txt requirements/dev.in
 #
--e git+https://github.com/kytos-ng/kytos.git#egg=kytos
+-e git+https://github.com/kytos-ng/kytos.git@feature/msg_out_pqueue#egg=kytos
```
Comment (viniarck) [inline requirements/dev.txt:7]: This commit 768acad9ec41325b2524ab85aee50313fb5ef843 will be reverted before getting merged, I'm using this to make sure the tests passes until the other PRs that this PR depends on get merged.

**Review by italovalcy — APPROVED** [review]
Looks good to me

File: requirements/dev.txt (hunk 1 above)
Comment (viniarck) [reply requirements/dev.txt]: I've just reverted it on [0304666](https://github.com/kytos-ng/flow_manager/pull/88/commits/0304666762ca0c22b45d67793b053670105b9711)

**Comment by viniarck** [comment]
> Looks good to me

Appreciated your review, Italo.

**Pull request merged**